lib/node_modules/@stdlib/math/base/special/cabsf/test/fixtures/julia/runner.jl

```diff
@@ -38,11 +38,8 @@ julia> gen( re, im, \"data.json\" );
 ```
 """
 function gen( re, im, name )
-	z = complex.( re, im );
-	y = Array{Float32}( undef, length(z) );
-	for i in eachindex(z)
-		y[i] = abs( ComplexF32( z[i] ) );
-	end
+ 	z = Complex{ Float32 }.( Float32.( re ), Float32.( im ) );
```

Comment:
Mixed tabs and spaces. And also, what is the advantage of this change? The previous code seems to be equivalent.